ledger/ledger-api-integration-tests/src/test/lib/scala/com/digitalasset/platform/PlatformApplications.scala

```diff
@@ -73,6 +71,19 @@ object PlatformApplications {
     def withMaxNumberOfAcsContracts(cap: Int) = copy(maxNumberOfAcsContracts = Some(cap))
 
     def withCommandConfiguration(cc: CommandConfiguration) = copy(commandConfiguration = cc)
+
+    def withRemoteApiEndpoint(endpoint: RemoteApiEndpoint) = copy(remoteApiEndpoint = Some(endpoint))
```

Comment:
looks good, but we should really couple the configs together with their relevant fixtures at some point, to avoid this common denominator style of über-configs

Reply:
Coming in future PRs!